src/bundle/Resources/views/content/tab/locations/tab.html.twig

```diff
@@ -51,7 +51,7 @@
                                     {% if not can_hide[location.id] %} disabled="disabled"{% endif %}
                                     value="{{ location.id }}"/>
                             </label>
-                            {% if location.contentInfo.isHidden or not is_location_visible %}
+                            {% if location.invisible or location.hidden %}
```

Comment:
According to https://github.com/ezsystems/ezplatform-kernel/blob/94937f8db831eb4229686f6cfde92e052a658b11/eZ/Publish/API/Repository/Values/Content/Location.php#L23:
> Indicates that the Location is not visible, being either marked as hidden itself, or implicitly hidden by its Content or an ancestor Location

`location.invisible` should be enough :wink: